dotnet/src/Functions/Functions.OpenApi.Extensions/Functions.OpenApi.Extensions.csproj

```diff
@@ -20,5 +20,6 @@
   <ItemGroup>
     <ProjectReference Include="..\..\SemanticKernel.Core\SemanticKernel.Core.csproj" />
     <ProjectReference Include="..\Functions.OpenApi\Functions.OpenApi.csproj" />
+    <ProjectReference Include="..\..\Agents\Abstractions\Agents.Abstractions.csproj"/>
```

Comment:
I'm curious on the broader thoughts on introducing this dependency relationship.  @markwallace-microsoft ?

Reply:
Depending on a *.Abstractions project is fine, this should pull in just the minimal Agent dependencies

Reply:
Great!

Reply:
so what I'm getting from this is that adding this reference is fine, and we should leave the new extension method where it is. Correct?